mteb/tasks/BitextMining/multilingual/WebFAQBitextMining.py

```diff
@@ -0,0 +1,1119 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskBitextMining import AbsTaskBitextMining
+from mteb.abstasks.MultilingualTask import MultilingualTask
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+_LANGUAGES = {  # Consider those 1001 language pairs with at least 100 samples
```

Comment:
This is probably too many, and many are probably unreasonable. 

We could consider Any to English. BibleNLPBitextMining does that. Flores uses Any to Any. However, it uses an efficient format that is only allowed because [everything is paired up](https://huggingface.co/datasets/mteb/flores). I don't think this is the case here.

Alternatively, we could use Glottolog and do it within language families (any to eng and also, e.g., a connection between all the Scandinavian languages). For a similar approach to this, take a look at NTREXBitextMining (it might be that you can just use this one).

How long does it take to run?